frontend/openchat-client/src/openchat.ts

```diff
@@ -4571,20 +4573,20 @@ export class OpenChat extends OpenChatAgentWorker {
         );
     }
 
-    buildUserLookupForMentions(includeSelf: boolean): Record<string, UserSummary> {
-        const map = {} as Record<string, UserSummary>;
-        const userStore = this._liveState.userStore;
-        const currentUser = this._user?.userId;
-        for (const member of this._liveState.currentChatMembers) {
-            const userId = member.userId;
-            if (includeSelf || userId !== currentUser) {
+    getUserLookupForMentions(): Record<string, UserSummary> {
```

Comment:
There doesn't seem to be any reason to return the lookup. Even simpler to just have a `lookupUserByUsername` function right?

Reply:
We need the full lookup to display the users in the mention picker

Reply:
Only because we don't have a `getUsersByUsernamePrefix` function on the openchat client. Just thinking that the more logic we can "hide" the better.